ultra/train.py

```diff
@@ -124,6 +124,12 @@ def train(task, num_episodes, policy_class, eval_info, timestep_sec, headless, s
         type=str,
         default="easy",
     )
+    parser.add_argument(
+        "--policy",
+        help="Policies available : [PPO, SAC, TD3, DQN]",
```

Comment:
Oh perhaps the help argument needs to be changed so that the stuff inside the brackets match available policies? Something like the following maybe?
```suggestion
        help="Policies available: [ppo, sac, ddpg, dqn, bdqn]",
```